h/templates/layouts/base.html

```diff
@@ -4,7 +4,7 @@
     {% block meta %}
       <meta charset="UTF-8" />
       <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1"/>
-      <meta name="viewport" content="width=device-width,initial-scale=1" />
+      <meta name="viewport" content="width=device-width,initial-scale=1,user-scalable=no" />
```

Comment:
> You sure we want to set this? It can be super annoying, especially for users who might have poor eyesight.

+1 to what @tilgovi has jusd said.

It's super annoying to work with apps in which some feature that I expect to be there (like zooming) is obviously artificially removed / disabled.